tools/block-generator/generator/teal/poap_boxes.teal

```diff
@@ -0,0 +1,504 @@
+// SOURCE: https://algoscan.app/app/946441472
```

Comment:
Introducing actual apps on the blockchain for better fidelity with real usage patterns.